src/exts/minigames.py

```diff
@@ -0,0 +1,103 @@
+from __future__ import annotations
+
+import random
+from typing import TYPE_CHECKING
+
+import disnake
+from disnake.ext import commands
+from src.views.shoot import ShootMenu
+
+if TYPE_CHECKING:
+    from src.bot import Universe
+    from src.views.shoot import ShootStats
+
+
+__all__: tuple[str, ...] = (
+    "generate_random_stats",
+    "Minigames",
+)
+
+
+def _generate_fake_mac() -> str:
+    tokens = "ABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789"
+    seq = random.choices(tokens, k=16)  # noqa: S311
+    return ":".join([f"{seq[i]}{seq[i+1]}" for i in range(0, len(seq), 2)])
+
+
+def generate_random_stats(stats: ShootStats, *, skip_health: bool = False) -> None:
+    for attr in stats.__slots__:
+        if attr in ("g_acc", "radians_angle", "ammunition", "angle", "total_shots", "hits"):
+            continue
+
+        if attr in ("enemy_position",):
+            x = int(stats.calculate_shot_range())
+            setattr(stats, attr, random.randint(x, x * 2 if x != 0 else 10))  # noqa: S311
+            continue
+
+        if attr in ("enemy_health",) and skip_health:
+            setattr(stats, attr, random.randint(1, 20))  # noqa: S311
+            continue
+
+        setattr(stats, attr, int(random.random() * 100))  # noqa: S311
+
+
+class Minigames(commands.Cog):
+    def __init__(self, bot: Universe) -> None:
+        self.bot = bot
+
+    # disnake typing skill issue
+    @commands.slash_command()  # type: ignore[reportUnknownMemberType]
+    async def shoot(self, inter: disnake.GuildCommandInteraction) -> None:
+        """Run an info overloaded shoot minigame."""
+        player = await self.bot.database.fetch_player(inter.author.id)
+        view = ShootMenu(inter.author, player)
+        generate_random_stats(view.stats)
+        embed = disnake.Embed(title="Shoot minigame", description="\n".join(["." * 10] * 5))
+        embed.add_field("Planet acceleration", f"{round(view.stats.g_acc, 2)} m/s^2")
+        embed.add_field("Position", f"{view.stats.position}")
+        embed.add_field("Angle", f"{view.stats.angle}")
+        embed.add_field("Ammunition (Shots left)", f"{view.stats.ammunition}")
+        embed.add_field("Energy (Moves left)", f"{view.stats.energy}")
+        embed.add_field("Ship Speed", f"{view.stats.ship_speed}")
+        embed.add_field("Bullet Velocity", f"{view.stats.bullet_velocity}")
+        embed.add_field("Bullet Type", f"{view.stats.bullet_type}")
+
+        embed.add_field("Obstacles in range", f"{view.stats.obstacles_in_range}")
+        embed.add_field("Fluff Stat X", f"{view.stats.fluff_stat_x}")
+        embed.add_field("Fluff Stat Y", f"{view.stats.fluff_stat_y}")
+        embed.add_field("Fluff Stat Z", f"{view.stats.fluff_stat_z}")
```

Comment:
These were temporary names, shouldn't they get renamed or removed?

Reply:
Since the goal behind these 4 fluff stats is to make the user think it could be important (contrary to all other useless informations) one could be replaced with something like the outerspace pressure (about 1.33E−11 Pa for ref). This value wont change but seems important. I don't have much more idea for the 3 other fluff values. Maybe we could drop them